modules/nodes-nodes-jobs-about.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-jobs.adoc
+
+[id='nodes-pods-configuring-replication_{context}']
+= About jobs 
+
+A job tracks the overall progress of a task and updates its status with information
+about active, succeeded, and failed pods. Deleting a job will clean up any pod
+replicas it created. Jobs are part of the Kubernetes API, which can be managed
+with `oc` commands like other object types.
+
+There are two types of jobs in {product-title}:
```

Comment:
This will be confusing. Jobs and CronJobs are distinct resources, even though CronJobs build on top of a Job, we can't say 'two types of jobs'.

Reply:
Myabe say there are two possible resource types that allow creating run-once objects?